pages/glossary/semantic-link.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Semantic link
+key: semantic-link
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - DOM tree
+---
+
+A semantic link is any HTML element which has the [semantic role][] of [`link`][link] or a [semantic role][] that inherits from the [`link`][link] role.
```

Comment:
Why only an HTML element? If we make this HTML and SVG, we can remove the use of that phrase from the applicability.

Reply:
Good point. I actually think the definition (and the rule) can be made technology agnostic since the concept s involved (role, name) are.